Editor/SmartIndent/XmlSmartIndent.cs

```diff
@@ -119,11 +119,23 @@ protected virtual int GetLineExpectedIndent (ITextSnapshotLine line, XmlBackgrou
 			//which were not closed by the end of the line
 			int depth = 0;
 
+			//special case if the line starts with something else than a closing tag,
+			//treat it as content and don't take the remaining closing tags on the
+			//current line into account
+			bool startsWithClosingTag = line.GetText ().StartsWith ("</");
```

Comment:
Shouldn't this skip leading whitespace? IIRC there are some utils for that. Also it can use Ordinal StartsWith (or index into the snapshot, IIRC that's more efficient than getting the line text)

Reply:
ah there we go: https://github.com/mhutch/MonoDevelop.Xml/blob/9723df17f821b4a3a04ba2534d1831e6ef429924/Editor/Roslyn/Extensions/ITextSnapshotLineExtensions.cs#L18

Reply:
Good point, fixed